### PR TITLE
Pc/arize

### PR DESCRIPTION
**Pull request by Blaizzy**
# Description

Include a summary of the changes and the related issue.

__Related to:__ <ClickUp/JIRA task name>

__Any expected test failures?__

---

Add a `[X]` to relevant checklist items

## ❔ This change

- [x] adds a new feature
- [ ] fixes breaking code
- [ ] is cosmetic (refactoring/reformatting)

---

## ✔️ Pre-merge checklist

- [x] Refactored code ([sourcery](https://sourcery.ai/))
- [x] Tested code locally
- [x] Precommit installed and run before pushing changes
- [x] Added code to GitHub tests ([notebooks](workflows/test-notebooks.yml), [scripts](workflows/test-scripts.yml))
- [x] Updated GitHub [README](../README.md)
- [ ] Updated the projects overview page on Notion

---

## 🧪 Test Configuration

- OS:
- Python version:
- Neptune version:
- Affected libraries with version:

### CODE REVIEW

**Comment by Blaizzy** [comment]
@SiddhantSadangi I've made the changes to scikit-learn.

File: integrations-and-supported-tools/arize/notebooks/Neptune_Arize.ipynb [hunk 1]
```diff
@@ -0,0 +1,237 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Arize integration guide\n",
+    "Arize and Neptune are MLOps tools that aim to improve connected but different parts of your ML pipeline and workflow.\n",
```
Comment (TytusKalicki) [inline integrations-and-supported-tools/arize/notebooks/Neptune_Arize.ipynb:8]: Arize and Neptune are MLOps tools that aim to improve different (but related) parts of your ML pipeline and workflow.

File: integrations-and-supported-tools/arize/notebooks/Neptune_Arize.ipynb [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+    "model_type = ModelTypes.BINARY_CLASSIFICATION\n",
+    "\n",
+    "\n",
+    "def process_data(X, y):\n",
+    "    X = np.array(X).reshape((len(X), 30))\n",
+    "    y = np.array(y)\n",
+    "    return X, y\n",
+    "\n",
+    "\n",
+    "# Load data and split data\n",
```
Comment (TytusKalicki) [inline integrations-and-supported-tools/arize/notebooks/Neptune_Arize.ipynb:92]: would "Load and split data" be correct as well? It sounds smoother

**Review by TytusKalicki — COMMENTED** [review]
sorry for the wait! looks good :)

**Review by TytusKalicki — APPROVED** [review]

**Pull request closed without merging**